Source/Details/_ASDisplayViewAccessiblity.mm

```diff
@@ -287,6 +287,12 @@ - (NSArray *)accessibilityElements
   NSMutableArray *accessibilityElements = [[NSMutableArray alloc] init];
   CollectAccessibilityElementsForView(self.view, accessibilityElements);
   SortAccessibilityElements(accessibilityElements);
+
+  if (_accessibilityElementsBlock) {
+    NSArray *accessibilityElementsCopy = [accessibilityElements copy];
+    accessibilityElements = _accessibilityElementsBlock(accessibilityElementsCopy);
```

Comment:
I'm wondering if it's ok to give the block a mutable array, so that the block may modify it directly and we don't need to copy it beforehand?

I'm not sure if it's a good API design?

Reply:
That would work too, I just felt this might be safer, sort of forcing people to create their own mutable array and add the object as needed and return a copy of it. Do we have an example of supporting custom blocks like this? it would be nice to follow an existing pattern too if there is one.